graylog2-server/src/main/java/org/graylog/schema/AssociatedFields.java

```diff
@@ -24,4 +24,5 @@
     public static final String ASSOCIATED_MAC = "associated_mac";
     public static final String ASSOCIATED_USER_ID = "associated_user_id";
     public static final String ASSOCIATED_USER_NAME = "associated_user_name";
+    public static final String ASSOCIATED_XFF_IP = "associated_xff_ip";
```

Comment:
This was on Traffic if I am reading it right, but I just forgot to move this one to http_xff, XLS is updated now.

Reply:
Ok, will remove this and shift Traffic line 75 to HTTP_XFF